tests/harvest/test_scenarios.py

```
@@ -167,11 +167,18 @@
     assert httpd_server_base
     for store in store_info_sets:
         log.debug(f"testing scenario one for {store}")
-        config = CONFIGS / "dereference_test2_sparql.yml"
-        travharv = service(
-            config,
-            store,
-        )
+        # config = CONFIGS / "dereference_test2_sparql.yml"
+
+        @pytest.fixture
+        def create_harvest_service(store_info_sets):
```

Ruff

Missing return type annotation for private function `create_harvest_service`

(ANN202)